io.github.Adda0.Stardrop.yaml

```diff
@@ -0,0 +1,76 @@
+app-id: io.github.Adda0.Stardrop
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet7
+build-options:
+    append-path: /usr/lib/sdk/dotnet7/bin
+    append-ld-library-path: /usr/lib/sdk/dotnet7/lib
+    env:
+        PKG_CONFIG_PATH: /app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet7/lib/pkgconfig
+modules:
+  # Taken from https://github.com/flathub/com.unity.UnityHub/blob/af171c1ee22994e7ef698004bd44ba7e8bc71cf0/openssl-1.1.yaml.
+  - name: openssl-1.1
+    buildsystem: simple
+    build-commands:
+      - ./config --prefix=/app
+      - make -j $FLATPAK_BUILDER_N_JOBS
+      - make install
+    cleanup:
+      - /share/doc
+      - /share/man
+      - '*.a'
+      # Don't mask CLI tools provided by runtime - we just want the versioned shared library
+      - /bin
+      # Don't mask unversion library symlinks from library
+      - /lib/libcrypto.so
+      - /lib/libssl.so
+    sources:
+      - type: archive
+        url: https://www.openssl.org/source/openssl-1.1.1w.tar.gz
+        sha256: cf3098950cb4d853ad95c0841f1f9c6d3dc102dccfcacd521d93925208b76ac8
+        # x-checker-data:
+        #   type: anitya
+        #   project-id: 20333
+        #   url-template: https://www.openssl.org/source/openssl-$version.tar.gz
+  - name: Stardrop
+    buildsystem: simple
+    build-commands:
+      - "install-sdk.sh"
+      - "dotnet publish -c Release --source ./nuget-sources Stardrop/Stardrop.csproj --runtime linux-x64 --self-contained true"
+      - cp -r /run/build/Stardrop/Stardrop/bin/Release/linux-x64/publish/* /app/bin/
+    sources:
+      - nuget-sources.json
+
+      - type: archive
+        url: "https://github.com/Adda0/Stardrop/archive/refs/tags/v1.0.0-beta.38-custom_adda.1.tar.gz"
+        sha256: eb1a31fa533ef0d3f4f95d49bb3f7ff1b6692307fb1d0aeb7c21b0b7debcad8d
+      # - type: dir
+      #   path: ../Stardrop-repo/
+  - name: Meta
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 io.github.Adda0.Stardrop.metainfo.xml /app/share/metainfo/io.github.Adda0.Stardrop.metainfo.xml
+      - install -Dm644 io.github.Adda0.Stardrop.desktop /app/share/applications/io.github.Adda0.Stardrop.desktop
+      - install -Dm644 Stardrop_512x512x32.png /app/share/icons/hicolor/512x512/apps/io.github.Adda0.Stardrop.png
+    sources:
+      - type: file
+        path: io.github.Adda0.Stardrop.metainfo.xml
+      - type: file
+        path: io.github.Adda0.Stardrop.desktop
+      - type: file
+        path: Stardrop_512x512x32.png
```

Comment:
these do not exist here? Also, desktop file, icon and appdata should come from the upstream, you should submit them.

Reply:
Just to clarify, by submitting, you just mean pushing all of them together with `nuget-sources.json` to this PR and the paths here in the manifest file remain the same, i.e., in the current directory. Is that right?

Reply:
Files are submitted in this PR.